### ISSUE TITLE
package not included in module namespace

### ISSUE THREAD

**Issue by DylanGriffith** [bitwalker/exprotobuf#23]
I'm not sure if this is a bug or feature request or if I'm just misunderstanding how this is supposed to work but in testing it seems like this library doesn't include the `package` name in the module namespace. I have the following test to explain what I mean:

```elixir
defmodule PackageNamespaceTest do
  use ExUnit.Case

  defmodule Proto do
    use Protobuf, """
      package TopLevel;
      message Msg {
        optional string name = 1;
      }
    """
  end

  test "it includes the package in the module name" do
    msg = Proto.TopLevel.Msg.new(name: "Ron")
    assert msg == %Proto.TopLevel.Msg{name: "Ron"}
  end
end
```

As an aside if this is not easy to make it work this way is there any underlying erlang code or something I can make use of for encoding/decoding based on the name `TopLevel.Msg`?

**Comment by DylanGriffith** [comment]
I have a working solution: https://github.com/bitwalker/exprotobuf/compare/master...DylanGriffith:with-package-name

It's pretty hacky as I am new to elixir, but happy to refactor if you have any suggestions.

**Comment by bitwalker** [comment]
@DylanGriffith Thanks for getting this working, I've glanced over the diff, but I'll have to give it a little more in depth though to address your comments. We can probably just pass the Config struct along, but that's without looking very closely at it.

**Comment by mmmries** [comment]
@DylanGriffith is this resolved by the work we did for `1.0.0.rc0`?

**Comment by DylanGriffith** [comment]
@mmmries Yes
